### PR TITLE
DONOG-91

### PR DESCRIPTION
**Pull request by elegier**

### CODE REVIEW

File: DonoGear/app/src/main/java/com/example/donogear/LoginPageActivity.java [hunk 1]
```diff
@@ -61,11 +84,69 @@ public void onTabReselected(TabLayout.Tab tab) {
...
+//                        startActivity(intent);
+                    }
+                });
+        AlertDialog ok = builder.create();
+        ok.show();
+    }
+
     public void createAccountBtnClicked(View view) {
 
+        username = usernameEditTextSignup.getText().toString();
```
Comment (elegier) [inline DonoGear/app/src/main/java/com/example/donogear/LoginPageActivity.java:188]: Need to add a null check for the user name and the password. Don't want the user keeping these fields blank

File: DonoGear/app/src/main/java/com/example/donogear/LoginPageActivity.java [hunk 2]
```diff
@@ -61,11 +84,69 @@ public void onTabReselected(TabLayout.Tab tab) {
...
+
+
+        user.signUpInBackground(new SignUpCallback() {
+            @Override
+            public void done(ParseException e) {
+                if (e == null) {
+                    alertDisplayer("Sucessful Sign Up!","Welcome" + username + "!");
+                } else {
+                    ParseUser.logOut();
+//                    Toast.makeText(SignUpActivity.this, e.getMessage(), Toast.LENGTH_LONG).show();
```
Comment (elegier) [inline DonoGear/app/src/main/java/com/example/donogear/LoginPageActivity.java:231]: I think the error should be logged here instead of using the toast

**Review by elegier — COMMENTED** [review]
-Need to resolve merge conflicts
-Need to add validation for username and password for signup case
-remove commented out/unneeded code for code smells

**Comment by FS1596** [comment]
@elegier I have made changes as suggested. Please verify and check if anything else is to be done

**Pull request merged**